docs/whats-new/dotnet-7-docs.md

```diff
@@ -0,0 +1,295 @@
+- [CA1422: Validate platform compatibility - obsoleted APIs](../fundamentals/code-analysis/quality-rules/ca1422.md)
+- [CA2260: Implement generic math interfaces correctly](../fundamentals/code-analysis/quality-rules/ca2260.md)
+- [CA1854: Prefer the `IDictionary.TryGetValue(TKey, out TValue)` method](../fundamentals/code-analysis/quality-rules/ca1854.md)
+- [CA1851: Possible multiple enumerations of `IEnumerable` collection](../fundamentals/code-analysis/quality-rules/ca1851.md)
+- [Prefer 'null' check over type check (IDE0150)](../fundamentals/code-analysis/style-rules/ide0150.md)
+- [Simplify property pattern (IDE0170)](../fundamentals/code-analysis/style-rules/ide0170.md)
+- [Use tuple to swap values (IDE0180)](../fundamentals/code-analysis/style-rules/ide0180.md)
+- [Namespace declaration preferences (IDE0160 and IDE0161)](../fundamentals/code-analysis/style-rules/ide0160-ide0161.md)
+- [.NET regular expression source generators](../standard/base-types/regular-expression-source-generators.md)
+- [Generic math](../standard/generics/math.md)
+- [Source generation for platform invokes](../standard/native-interop/pinvoke-source-generation.md)
+- [Source generation for custom marshalling](../standard/native-interop/custom-marshalling-source-generation.md)
+- [Tutorial: Use custom marshallers in source-generated P/Invokes](../standard/native-interop/tutorial-custom-marshaller.md)
+- [Customize a JSON contract](../standard/serialization/system-text-json/custom-contracts.md)
+
+### Updated articles
+
+- [Native AOT Deployment](../core/deploying/native-aot/index.md) - Document AOT analysis warnings
+- [Single-file deployment and executable](../core/deploying/single-file/overview.md) - Update single file incompatibility docs
+- [Prepare .NET libraries for trimming](../core/deploying/trimming/prepare-libraries-for-trimming.md)- Update trimming libraries docs
+- [Trimming options](../core/deploying/trimming/trimming-options.md) - Cleanup trimming options and document TrimMode=full,partial
+- [Tutorial: Containerize a .NET app](../core/docker/build-container.md) - Add `dotnet publish` article for .NET 7 containerization support.
+- [Rate limit an HTTP handler in .NET](../core/extensions/http-ratelimiter.md) - New `RateLimiter` article showing a custom HTTP handler
+- [Configuration providers in .NET](../core/extensions/configuration-providers.md) - Environment variable prefix breaking change
+- [Use HTTP/3 with HttpClient](../core/extensions/httpclient-http3.md) - Update HTTP/3 support for HttpClient in .NET 7
+- [IHttpClientFactory with .NET](../core/extensions/httpclient-factory.md) - New `HttpClient` article
+- [Overview of .NET, MSBuild, and Visual Studio versioning](../core/porting/versioning-sdk-msbuild-vs.md)
+  - Add preview SDK table
+  - Formalize versioning support policy
+- [Overview of the .NET Upgrade Assistant](../core/porting/upgrade-assistant-overview.md) - Merge .NET 7 release branch into main
```

Comment:
Should this be included?  And should it have a different note about the change?

Reply:
No, this should be removed. It wasn't updated for .NET 7.